### DISCUSSION TITLE
Is it possible to turn off all animations during e2e testing?

### DISCUSSION THREAD

**Discussion by Hiroshiba** [quasarframework/quasar#16160] [CLI - Electron mode]
I am developing software using Quasar and recently started using Playwright to introduce e2e testing. Since Quasar appropriately uses roles and IDs, I can effectively utilize automated test generation, making the whole process very stress-free. However, during these e2e tests, there are many instances where it's better to wait if there are any animations or transitions occurring. I was looking for a way to turn off all animations just during testing, but couldn't find one. Is there a method to do this within Quasar?

**Comment by joshas** [comment]
Have you tried enabling [reducedMotion](https://playwright.dev/docs/api/class-browser#browser-new-context-option-reduced-motion) preference on Playwright? Not sure if Quasar fully supports `prefers-reduced-motion` media feature, but it would be worth a try.

**Reply by Hiroshiba** [reply]
Thank you!
I tried inserting this code:

```ts
await page.emulateMedia({ reducedMotion: "reduce" });
```

I gave it a shot, but it seems like the animations didn't completely disappear. I'm not too familiar with Playwright, so I might be drawing the wrong conclusion though...
